### ISSUE TITLE
Seeding Random dropout

### ISSUE THREAD

**Issue by jimmyroyer** [aigamedev/scikit-neuralnetwork#203]
Hello, is it possible to seed (initialize) the random dropout regularization in the MLP classifier?
Thanks

**Comment by alexjc** [comment]
I presume you tried `seed` as constructor argument and it's not deterministic?

**Comment by jimmyroyer** [comment]
Seed works but it seeds the whole function. For instance if I use a randomized search for the meta parameters it will seed the searched parameters as well as the dropouts and the weights. I was wondering if it is possible to only seed the random dropout part of the classifier.

**Comment by alexjc** [comment]
No, it's not currently possible. Could you share why you need dropout-only seed?

**Comment by jimmyroyer** [comment]
I want to use the seed of the weights and the randomized search as another "hyper" parameter.

**Comment by alexjc** [comment]
I'm not sure I would support dropout as separate seed, it shouldn't make any difference as the iterations increase. More important is to expose the weight initialization strategy!
